app/models/transformation_mapping/vm_migration_validator.rb

```diff
@@ -128,6 +133,26 @@ def mapped_lans
     @mapped_lans ||= Lan.where(:id => @mapping.transformation_mapping_items.where(:source_type => 'Lan').select(:source_id))
   end
 
+  # rubocop: disable Naming/UncommunicativeMethodParamName
```

Comment:
Just ignore rubocop issues that you don't think make sense rather than adding these diable comments, it makes the code more cluttered and harder to read which is the opposite intention of us using rubocop.  Getting to 0 rubocop comments isn't something we as mergers look at, we only look at if the rubocop issues make sense

Reply:
Done

Reply:
:+1: